docs/scos/dev/feature-integration-guides/202307.0/glue-api/data-exchange-api/install-the-data-exchange-api.md

```diff
@@ -186,9 +213,11 @@ Ensure the following transfers have been created:
 
 {% endinfo_block %}
 
+2. Add configurations for dynamic entities. For instructions, see [How to configure Data Exchange API](/docs/scos/dev/glue-api-guides/{{page.version}}/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.html).  
```

Comment:
1. Is this step obligatory to install the API?
2. Is it possible to do this step before I start the installation of the API?

Reply:
1. No, it's mostly about the usage of it.
2. Definitely not